transports/upgrades/rabbitmq-classic-to-quorum-migration.md

```diff
@@ -0,0 +1,90 @@
+
+1. Update the instance to version 4.23.0.
+1. Under **Installation Path**, click **Browse**.
+1. In the Explorer window that appears, locate and edit the **ServiceControl.exe.config** file.
+1. Change the value of the `ServiceControl/TransportType` setting to `ServiceControl.Transports.RabbitMQ.RabbitMQQuorumConventionalRoutingTransportCustomization, ServiceControl.Transports.RabbitMQ`.
+1. In ServiceControl Management Utility, start the instance.
+
+### Step 7: Update endpoints
+
+Update each endpoint to use NServiceBus.RabbitMQ 7.0.0. As part of this upgrade, set the [queue type](/transports/rabbitmq/routing-topology.md#controlling-queue-type) to use classic queues.
+
+The goal of this upgrade is to migrate delayed messages to the new delay infrastructure. Updating the endpoint's input queues to use quorum queues can come later.
+
+## Step 8: Migrate delayed messages
+
+Starting with NServiceBus.RabbitMQ version 7.0.0, a new version of the delay infrastructure based on quorum queues is created, which ensures that delayed messages cannot be lost in the event of a network partition.
+
+* The "v1" infrastructure contains 27 queues with the pattern `nsb.delay-level-##`, which use classic queues
+* The "v2" infrastructure contains 27 queues with the pattern `nsb.v2.delay-level-##` which use quorum queues.
+
+The two delay infrastructures can be used in parallel for as long as is necessary, but delayed messages migrated to the v2 delay infrastructure are safer than in the v1 infrastructure.
+
+Use the [`delays migrate`](/transports/rabbitmq/operations-scripting.md#delays-migrate) command provided by the command line tool to move all existing delayed messages to the v2 delay infrastructure. This command can be safely run multiple times until all endpoints are updated to version 7.0.0, and the tool confirms that all delayed messages have been moved to the new infrastructure.
+
+Delayed message migration can be done after each endpoint is upgraded, or only once at the end of the process, after every endpoint is updated to version 7.0.0. Messages that are migrated are safe from network partitions, however, as long as there are still endpoints that have not been updated to version 7.0.0, it's possible for additional messages to be delivered to the v1 delay infrastructure. This is why the tool must be run at the very least after all endpoints are migrated.
+
+## Step 9: Remove v1 delay infrastructure
+
+After all delayed messages have been migrated, the exchanges and queues that make up the v1 delay infrastructure (queues and exchanges matching `nsb.delay-level-##` and the `nsb.delay-delivery` exchange) can be removed from the broker.
+
+### Step 10: Migrate endpoints
+
+Endpoints can now be migrated to use quorum queues as their input queues on a per-endpoint basis. 
+
+For each endpoint:
+
+1. In the endpoint configuration code, set the [queue type](/transports/rabbitmq/routing-topology.md#controlling-queue-type) to use quorum queues.
```

Comment:
This could be a little misleading, if you update your endpoint configuration code to quorum and restart prior to the next step, you'll get errors from the installers.  It may be more clear to have the steps be:
1.) Shutdown the endpoint to be migrated
2.) Use the `queue migrate-to-quorum`...
3.) Update endpoint config
4.) Deploy